### PR TITLE
Update charmap.txt with more comments

### PR DESCRIPTION
**Pull request by FieryMewtwo**

### CODE REVIEW

File: charmap.txt [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // Character mapping for Pokémon HeartGold and SoulSilver
-// Version 2021.08.17
-// Format is HEXCODE=<character> or HEXCODE={<command>}
```
Comment (PikalaxALT) [inline charmap.txt:3]: Restore this line

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

File: charmap.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+// Version 2021.09.22
```
Comment (PikalaxALT) [inline charmap.txt:2]: Version bump not necessary

File: charmap.txt (hunk 1 above)
Comment (FieryMewtwo) [reply charmap.txt:3]: Sure thing

**Comment by FieryMewtwo** [comment]
Okay, done.

**Comment by PikalaxALT** [comment]
Looking at this again, I fail to see what value this brings to the project. Closing.

**Pull request closed without merging**